tbtc-ts/src/sweep.ts

```diff
@@ -0,0 +1,107 @@
+// @ts-ignore
+import bcoin from "bcoin"
+// @ts-ignore
+import wif from "wif"
+import { BigNumber } from "ethers"
+import {
+  RawTransaction,
+  UnspentTransactionOutput,
+  Client as BitcoinClient,
+} from "./bitcoin"
+
+/**
+ * Sweeps UTXOs by combining all the provided UTXOs and broadcasting a Bitcoin
+ * P2WPKH sweep transaction.
+ * @dev The caller is responsible for ensuring the provided UTXOs are correctly
+ *      formed, can be spent by the wallet and their combined value is greater
+ *      then the fee.
+ * @param utxos - UTXOs to be combined into one output.
```

Comment:
There should be also the UTXO from the previous sweep. Worth noting that in all TSDocs.